pkg/apis/flows/register.go

```diff
@@ -0,0 +1,21 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package flows
+
+const (
+	GroupName = "flows.knative.dev"
```

Comment:
register them

https://github.com/knative/eventing/blob/master/cmd/webhook/main.go#L75-L79

?

Reply:
I just see that this PR is just types.